src/journal/views.py

```diff
@@ -133,6 +133,11 @@ def serve_journal_cover(request):
 
 @has_journal
 def funder_articles(request, funder_id):
```

Comment:
Consider: Should we mark this view as deprecated? I don't think its particularly useful.